### ISSUE TITLE
Menu cannot be closed if opened from within another menu

### ISSUE THREAD

**Issue by nathangath** [FrDH/jQuery.mmenu#619]
If the trigger for Menu X is located within Menu Y, Menu X will open but the PageDim will not fire and you will not be able to close Menu X by clicking outside of it.

http://codepen.io/nathangath/pen/xEKLPv

**Comment by FrDH** [comment]
Thank you for reporting this issue. I think this should be able to work with some debugging, I'll keep you posted.
For now, a solution would be to use a timeout to manually open the menu.

**Comment by nathangath** [comment]
Thanks. I was able to make it work by manually calling the open and close functions and then adding the relevant classes to the HTML element.

```
$(".js-opensecondmenu").click(function() {
    firstAPI.close(); 
    secondAPI.open();
});

secondAPI.bind('opened', function () {
    $('html').addClass('mm-opened mm-blocking mm-background mm-pagedim-black');
});
```

**Comment by FrDH** [comment]
Found the issue, will be fixed in the next release.

**Comment by FrDH** [comment]
Should be fixed in the new version 5.7.4

**Comment by nathangath** [comment]
Fantastic. Got it working in my project 👍
